### DISCUSSION TITLE
Limit query by url link

### DISCUSSION THREAD

**Discussion by thowt** [rappasoft/laravel-livewire-tables#569] [Q&A] [answered]
Hi guys,

im following docs, but probably i missed it somewhere…
I have working example (by docs) and what i am trying to achieve now is simple limit query “by url”. 

Lets say for example, we have o route /invoice/list/{customer_id}
this will return datatable of invoices related to customer by id. 
So i expect to pass customer_id somewhere in component to where, but with no success. 

any idea? Thank you.

**Comment by rappasoft** [comment, marked as answer]
Not a package feature just regular Livewire: https://laravel-livewire.com/docs/2.x/rendering-components#parameters

**Reply by thowt** [reply]
Hello,

thank you for response. I mean, i know how to pass it to component, but not sure how to modify 
public function query()
{
  return Invoice::query();
}

i get something with ->when and getFilter but that could be modified in frontend by user, because it shows in datatable header as active filter. I used something similar to Advanced Example in docs. 

Thank you.

**Reply by rappasoft** [reply]
You would set it as a property of the table and then use it:

```html
<livewire:datatable :customerId="request('customer_id')" />
```

```php
public $customerId;

...

public function query()
{
    return Invoice::where('customer_id', $this->customerId);
}
```

Or something like that.

**Reply by thowt** [reply]
Omg! 💯  
I put where and query together. I forgot to remove query 🤦‍♂️ 
Thank you for help.
